redisinsight/ui/src/slices/tests/rdi/pipeline.spec.ts

```diff
@@ -10,11 +10,15 @@ import reducer, {
   deployPipeline,
   deployPipelineSuccess,
   deployPipelineFailure,
+  getPipelineStrategies,
+  getPipelineStrategiesSuccess,
+  getPipelineStrategiesFailure,
+  setPipelineSchema,
   fetchRdiPipeline,
   deployPipelineAction,
   rdiPipelineSelector,
   fetchRdiPipelineSchema,
-  setPipelineSchema,
+  fetchPipelineStrategies, fetchPipelineTemplate,
```

Comment:
small nitpick - could you put these on two separate lines?

Reply:
For sure. done :)